index.js

```diff
@@ -4,6 +4,14 @@ var mktemp = require('mktemp')
 var rimraf = require('rimraf')
 var underscoreString = require('underscore.string')
 
+exports.buildTmpPath = buildTmpPath
+function buildTmpPath(obj, prop, className) {
+  if (className == null) className = obj.constructor && obj.constructor.name
+
+  var tmpDirName = prettyTmpDirName(className, prop)
+  return obj[prop] = path.join(findBaseDir(), tmpDirName)
```

Comment:
ya, I think some other packages use it for the shear convenience value (i.e. `ember-cli` uses it in its test suite)